rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -284,6 +284,99 @@ bool game_list_frame::IsEntryVisible(const game_info& game, bool search_fallback
 	return is_visible && matches_category() && SearchMatchesApp(qstr(game->info.name), serial, search_fallback);
 }
 
+bool game_list_frame::RemoveContentPath(const std::string& path, const std::string& desc)
+{
+	if (fs::exists(path))
```

Comment:
We mostly use early returns, which mostly makes the code much more readable and reduces indentations.
```suggestion
	if (!fs::exists(path))
	{
		return true;
	}
```